src/Microsoft.AspNetCore.Identity.UI/ConfigureApplicationCookieAreaSetup.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    internal class ConfigureApplicationCookieAreaSetup : IConfigureNamedOptions<CookieAuthenticationOptions>
+    {
+        private static readonly string[] IdentityCookieSchemes = new[]
+        {
+            IdentityConstants.ApplicationScheme,
+            IdentityConstants.ExternalScheme,
+            IdentityConstants.TwoFactorRememberMeScheme,
+            IdentityConstants.TwoFactorUserIdScheme
+        };
+
+        public void Configure(string name, CookieAuthenticationOptions options)
+        {
+            if (Array.IndexOf(IdentityCookieSchemes, name) != -1)
```

Comment:
This doesn't look right.  This should be a no-op if name != IdentityConstants.ApplicationScheme, you should just ignore the other constants.